tests/system/provisioning/one_manager_agent/roles/manager-role/tasks/main.yml

```diff
@@ -79,7 +79,10 @@
 - name: Start Wazuh
   command: /var/ossec/bin/wazuh-control restart
 
-- name: Create script to check BD
+- name: Copy wdb-query.py script
   copy:
-    src: ../../../../../scripts/system_query_db.py
-    dest: /var/
+    src: ../../../../scripts/wdb-query.py
+    dest: /var/ossec/bin/wdb-query.py
+    owner: root
+    group: root
+    mode: '0644'
```

Comment:
```suggestion
    mode: '0644'
    
```

Reply:
Added in https://github.com/wazuh/wazuh-qa/pull/5089/commits/e13544c3c3b5e7fb952b25aca5aeefc7db54a659